CHANGELOG.md

```diff
@@ -0,0 +1,178 @@
+- Fix ambiguity in queryDispatch\(\) overload resolution [\#293](https://github.com/arborx/ArborX/pull/293)
+- Properly update hash in version file when building from subdirs [\#266](https://github.com/arborx/ArborX/pull/266)
+- Avoid second pass for radius search when the results are empty [\#240](https://github.com/arborx/ArborX/pull/240)
+- Avoid more compiler warnings for nvcc\_wrapper [\#185](https://github.com/arborx/ArborX/pull/185)
+- Suppress Thrust warnings [\#178](https://github.com/arborx/ArborX/pull/178)
+- Fix segfault in Distributor [\#296](https://github.com/arborx/ArborX/pulls/296)
+- Allow non device type template parameter for output views in BVH::query\(\) [\#271](https://github.com/arborx/ArborX/pull/271)
+
+## [v0.8-beta2](https://github.com/arborx/arborx/tree/v0.8-beta2) (2019-10-10)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/97bbec21cc92dd2b4bd3a68c52a230b4c3c4643c...v0.8-beta2)
+
+**New features:**
+
+- Provide access traits for predicates [\#117](https://github.com/arborx/ArborX/pull/117)
+- Add version\(\) and gitCommitHash\(\) [\#61](https://github.com/arborx/ArborX/pull/61)
+
+**Build changes:**
+
+- Replace TRIBITS build system with CMake [\#14](https://github.com/arborx/ArborX/pull/14)
+- Replace ArborX\_ENABLE with ARBORX\_ENABLE [\#99](https://github.com/arborx/ArborX/pull/99)
+- Provide an option to disable tests and examples [\#31](https://github.com/arborx/ArborX/pull/31)
+- Make MPI dependency optional [\#42](https://github.com/arborx/ArborX/pull/42)
+
+**Enhancements:**
+
+- Use MPI\_Comm\_dup to separate ArborX comm context from user's [\#135](https://github.com/arborx/ArborX/pull/135)
+- Add CMake option for enabling benchmarks [\#138](https://github.com/arborx/ArborX/pull/138)
+- Add intersects\(Point, Box\) [\#122](https://github.com/arborx/ArborX/pull/122)
+- Mark BVH::{size,empty,bounds} as noexcept [\#114](https://github.com/arborx/ArborX/pull/114)
+- Improve error messages in BVH constructor and BVH::query\(\) [\#113](https://github.com/arborx/ArborX/pull/113)
+- Make data private in Point [\#100](https://github.com/arborx/ArborX/pull/100)
+- Constexpr geometric primitives and algorithms [\#97](https://github.com/arborx/ArborX/pull/97)
+- Add CUDA+clang CI test [\#95](https://github.com/arborx/ArborX/pull/95)
+- Make sure the recorded git hash is recent [\#79](https://github.com/arborx/ArborX/pull/79)
+- Test installation of ArborX [\#48](https://github.com/arborx/ArborX/pull/48)
+- Relax CudaUVM requirement [\#24](https://github.com/arborx/ArborX/pull/24)
+- Find Boost in subdirectories that actually require it [\#22](https://github.com/arborx/ArborX/pull/22)
+
+**Fixed bugs:**
+
+- Fix parsing options in the ArborX benchmark [\#54](https://github.com/arborx/ArborX/pulls/54)
+- Fix compilation against Kokkos development branch [\#40](https://github.com/arborx/ArborX/pulls/40)
```

Comment:
```suggestion
```

Reply:
I would revert this, I find it useful, especially the second one.

Reply:
It makes little sense, especially for the initial release